src/components/menu/examples/menu-subitems.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+    MenuItem,
+    ListSeparator,
+    LimelMenuCustomEvent,
+} from '@limetech/lime-elements';
+import { Component, State, h } from '@stencil/core';
+
+/**
+ * With Sub items
+ */
+@Component({
+    tag: 'limel-example-menu-subitems',
+    shadow: true,
+})
+export class MenuSubItemsExample {
+    private items: Array<MenuItem | ListSeparator> = [
+        {
+            text: 'Format',
```

Comment:
![image](https://github.com/Lundalogik/lime-elements/assets/35954987/8d41616f-dd3e-41f1-a8cb-07ddb8bbe036)
There is an empty icon in this 🤔

Reply:
same in `menu-custom-navigation.tsx`

Reply:
@BjornOstberg 's fix here: https://github.com/Lundalogik/lime-elements/pull/2550